mava/networks.py

```diff
@@ -166,8 +177,8 @@ def __call__(self, observation: Observation) -> tfd.Distribution:
         return self.action_head(obs_embedding, observation)
 
 
-class FeedForwardCritic(nn.Module):
-    """Feedforward Critic Network."""
+class FeedForwardValueNet(nn.Module):
```

Comment:
Nit: 
One more detail here, can we please rename `RecurrentCritic` -> `RecurrentValueNet` as well for consistency?